app_bikes/settings.py

```diff
@@ -0,0 +1,3 @@
+VALUES_UPDATE_INTERVAL = 10
+STAR_API_KEY = ''
+VELIB_API_KEY = ''
```

Comment:
Empty keys work when developing, not sure we should store api_keys as they are personal data.